### DISCUSSION TITLE
Different IPs for multiple containers

### DISCUSSION THREAD

**Discussion by huydinh3010** [qdm12/gluetun#1223] [Q&A]
My situation:
Im running 5 to 10 Gluetun containers simultaneously. I want to make sure that they always have different Endpoint IPs at the same REGION (Vietnam). One way is to configure the SERVER_HOSTNAME differently, but im using "private internet access" service, they have only one Vietnam's hostname is "vietnam.privacy.network". 
What is the best solution for my situation?
Thanks!

**Comment by qdm12** [comment]
You can use VPN_ENDPOINT_IP to set the target ip address. You can lookup `vietnam.privacy.network` in the file https://raw.githubusercontent.com/qdm12/gluetun/master/internal/storage/servers.json to find all the ips you can use.

**Reply by huydinh3010** [reply]
How can I find more Vietnam's VPN_ENDPOINT_IPs? Some IPs in that file are quite slow.

**Reply by qdm12** [reply]
You can try https://github.com/qdm12/gluetun/wiki/Updating-servers to get updated IPs (if any). Other than that, the problem would be on PIA's end.
